pressbooks/pressbooks-book
# Create new CTA for webbook pages

Fix for https://github.com/pressbooks/private/issues/884

To test: 
1. checkout this branch and visit a book homepage. You should see the CTA banner displayed in the networks' primary foreground/background colors. The link should take you to a page that's currently 404 but will be coming soon on marketing site.
2. Visit the internal pages of the book. You should also see the CTA banner on each of these pages.
3. Appearance of the reading interface and book homepage should appear normal (no weird extra spacing added or lost).
